configs/defconfig.esp32s3

```diff
@@ -7,4 +7,5 @@ CONFIG_ESP32S3_UNIVERSAL_MAC_ADDRESSES_TWO=y
 # CONFIG_ESP_TASK_WDT_CHECK_IDLE_TASK_CPU1 is not set
 CONFIG_FREERTOS_WATCHPOINT_END_OF_STACK=y
 CONFIG_FREERTOS_TIMER_TASK_STACK_DEPTH=3120
-CONFIG_ESP_SYSTEM_MEMPROT_FEATURE=n
\ No newline at end of file
+CONFIG_ESP_SYSTEM_MEMPROT_FEATURE=n
+CONFIG_BT_BLE_BLUFI_ENABLE=y
```

Comment:
This should actually go into `defconfig.common`, so it is enabled for all chips that have BLE